Sources/MapboxNavigation/VisualInstructionComponent.swift

```diff
@@ -13,7 +13,7 @@ extension VisualInstruction.Component {
             return imageRepresentation.legacyCacheKey ?? "generic-" + alternativeText.text
         case .text, .delimiter, .lane:
             return nil
-        case .guidanceView(let guidanceViewRepresentation, _):
+        case .guidanceView(let guidanceViewRepresentation, _, _):
             guard let imageURL = guidanceViewRepresentation.imageURL else { return nil }
             return "guidance-" + imageURL.absoluteString
```

Comment:
Should the cache key incorporate the kind of guidance view, or will the image URL be enough to avoid collisions?

Reply:
It's not critical to have subtype as part of cache key, so will leave it the way it is. BTW this updated signature of `VisualInstruction.Component.guidanceView()` is a breaking change, do not see good options to prevent it.